_includes/v19.2/misc/session-vars.html

```diff
@@ -307,6 +307,20 @@
    <td>Yes</td>
   </tr>
 
+  <tr>
+   <td>
+    <code>vectorize</code>
+   </td>
+   <td>The vectorized execution engine mode. Options include <code>auto</code>, <code>experimental_on</code>, and <code>off</code>.
+     For more details, see <a href="vectorized-execution.html#configuring-vectorized-execution">Configuring vectorized execution for CockroachDB</a>.
```

Comment:
Nice!  Glad you wrote up separate configuration instructions, would have been too complex here.